iree/compiler/Conversion/Common/RemoveDeadMemAllocsPass.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Google LLC
```

Comment:
We are in 2021 now. :p
(I'm not pretty sure if we update license in new files to 2021)

Reply:
I changed it to 2019. I think we have a script for license headers?

Reply:
We have a script to generate the cmakelist file with copyright. If the cmakelist file already has copyright, it will be ignored. Otherwise, the copyright will be `datetime.date.today().year`. Thus I think all new files should use the current year. But I guess it is not a big deal, I actually don't really know the convention about this.

Reply:
Year should match when the file was created (which could be different from when the file was committed to a branch). Do not update the year when moving or copying a file. Do not update the year when the year changes.